lib/usd/ui/assetResolver/AssetResolverUtils.h

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2025 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef MAYAUSDUI_USD_ASSETRESOLVERUTILS_H
+#define MAYAUSDUI_USD_ASSETRESOLVERUTILS_H
+
+#include <mayaUsd/mayaUsd.h>
+#include <mayaUsdUI/ui/api.h>
+
+#include <maya/MStatus.h>
+
+namespace MAYAUSD_NS_DEF {
+
+/** \class AssetResolverUtils
+\brief A utility class for managing Autodesk USD Asset Resolver settings in Maya.
+*/
+class MAYAUSD_UI_PUBLIC AssetResolverUtils
+{
+public:
+    // Include Maya project tokens in the Autodesk USD Asset Resolver
+    static void IncludeMayaProjectTokensInAdskAssetResolver();
+    // Exclude Maya project tokens from the Autodesk USD Asset Resolver
+    static void ExcludeMayaProjectTokensFromAdskAssetResolver();
```

Comment:
Technically according to our coding standard, the naming should use lowerCamelCase (start with lowercase). You did this in the other .h above.